### DISCUSSION TITLE
PostgreSQL 17?

### DISCUSSION THREAD

**Discussion by bjorntj** [keycloak/keycloak#33493] [Archive]
Hi.

I tried moving Keycloak 25.0.6 to the new PostgreSQL 17 but that kind of did not work at all. Do you need to explicitly add support for PostgreSQL 17?

I am getting the following error:

ERROR: Failed to obtain JDBC connection
ERROR: Driver does not support the provided URL: jdbc:postgresql://127.0.0.1/keycloak

**Comment by jonkoops** [comment]
PostgreSQL 17 is not supported (or at least officially tested). You can find a list of [supported databases](https://www.keycloak.org/server/db) on our website. I suggest that you file an issue if you would like to request support, please make sure to include any details on what does and does not work.

**Reply by bjorntj** [reply]
Oki.. But a bit hard to test at the moment, since the keycloak application won't event start when using PostgreSQL 17..
